submissions/blueprint-rant/blueprint-rant.md

```diff
@@ -0,0 +1,51 @@
+- Rseding91
+
+### The goal of Factorio
+
+The goal of Factorio is, *spoiler alert*, to build a factory, but it's so much more than that. Its about optimizing and improving your production setups, whether it be to produce more of the final product, make the bare minimum required in the tiniest space possible or keep that UPS up at 60 while allow you to keep building onwards. Each and every time you build, a green circuit maker for example, you get to use your experience to make it better than the last. Maybe this time you'll use productivity modules? Maybe you wan't to try centralized green circuit production this time round, but only have basic tiers of machines. This is the challenge of Factorio, designing within the constraints of the situation and making the best of it.
+
+![](Green circuit maker screenshot)
+*A simple but large footprint green circuit maker I recently designed to allow the production of 2 belts of green circuits, using only low tier assemblers. Notice the unique and slightly odd design, because why not?.*
+
+By just importing a blueprint every time you need to make something new, you aren't problem solving and designing, you are a construction bot blinding placing racks of machines, and plumbing up the inputs and outputs. That to me really doesn't sound very fun. It sounds like you're playing the fun*ish* in-game scenario, transport belt madness rather than pursuing your own destiny in freeplay.
+
+### Unless, blueprint importing is good?
+
+I will concede, in some **RARE** situations, importing blueprints can be handy to avoid several minutes of monotony. The perfect example of this is solar setups, generally tiling masses containing solar panels and accumulators, in the hope of one day cleanly powering your factory. Spending 5-10 minutes faffing around, trying to perfectly fill up the space around one roboport may sound like fun to some people, but this is a problem that doesn't particularly change depending on the game scenario. Whether you have just researched solar panels or are several hundred hours into maximising the SPM of a save, your solar panel setups are gonna look basically the same, with probably the main difference being the size of the blueprint placed at a single time.
+
+![](Generic tiling Solar Array blueprint)
+
+Balancers, *the black magic creations*, are another example of something that most players want to use, with fair reason, but don't want to learn and understand the maths and mechanics behind them. Figuring out the exact combination of belts, splitters and undergrounds to make the ratios perfect make sounds like a fantastic sunday afternoon for some people, but many would find it dull and unexciting. In both these scenarios, allowing the importing of blueprints will *likely* improve the players enjoyment of the game as they wont get stuck making monotonous blueprints or diving into the maths of balancers.
+
+![](Factorio balancer overview)
+
+### Blueprint library and copy/paste
```

Comment:
I don't think this should be a new section. Rather combine it with the previous one.